simple_commands.py

```diff
@@ -126,7 +126,7 @@ def stats(bot, update):
         )
 
         n = us.first_places
-        m = round((us.first_places / us.games_played) * 100)
+        m = round((us.first_places / us.games_played) * 100) if games_played else 0
```

Comment:
of course we still need the check :see_no_evil: but it should be like this right?

```suggestion
        m = round((us.first_places / us.games_played) * 100) if us.games_played else 0
```

Reply:
I'm clearly experiencing the consequences of too much uno playing and probably lack of caffeine